types/amulet/20210221.d.ts

```diff
@@ -0,0 +1,43 @@
+/* tslint:disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+/**
+ * This schema describes version 20210221 of the metadata standard for the amulet poem format, designed for the Zora protocol.
+ */
+export interface Amulet20210221 {
+  /**
+   * An HTTPS link to the carbon offset purchased in this poem's name. This is not required in this metadata version, but if it is not specified in this field, it should be included in the `description` field.
+   */
+  carbonOffsetURL?: {
+    [k: string]: unknown;
```

Comment:
Should this just be a `string`?

Reply:
Ah; that's the autogenerated code and I believe it autogenerated itself that way because I forgot to specify the `type` of the carbonOffsetURL in the JSON schema. Fixed!